File: solutions/csharp/raindrops/1/Raindrops.cs
using System;
using System.Text;
public static class Raindrops
{
    public static string Convert(int number)
    {
        StringBuilder output = new StringBuilder("");

        if (number % 3 == 0)
        {
            output.Append("Pling");
        }
        if (number % 5 == 0)
        {
            output.Append("Plang");
        }
        if (number % 7 == 0)
        {
            output.Append("Plong");
        }
        if (number % 3 != 0 && number % 5 != 0 && number % 7 != 0)
        {
            output.Append(number.ToString());
        }

        return output.ToString();
    }
}

File: solutions/javascript/raindrops/1/raindrops.js
//
// This is only a SKELETON file for the 'Raindrops' exercise. It's been provided as a
// convenience to get you started writing code faster.
//

export const convert = num => {
  if (num % 7 === 0 && num % 5 === 0 && num % 3 === 0) {
    return 'PlingPlangPlong';
  }
  if (num % 5 === 0 && num % 3 === 0) {
    return 'PlingPlang';
  }
  if (num % 3 === 0 && num % 7 === 0) {
    return 'PlingPlong';
  }
  if (num % 5 === 0 && num % 7 === 0) {
    return 'PlangPlong';
  }
  if (num % 7 === 0) {
    return 'Plong';
  }
  if (num % 5 === 0) {
    return 'Plang';
  }
  if (num % 3 === 0) {
    return 'Pling';
  }
  if (num % 7 !== 0 || num % 5 !== 0 || num % 3 !== 0) {
    return num.toString();
  }
}